docs/resources/Channel.md

```diff
@@ -901,7 +901,7 @@ You may create a message as a reply to another message. To do so, include a [`me
 | content           | string                                                                                            | the message contents (up to 2000 characters)                 | one of content, file, embed |
 | tts               | boolean                                                                                           | true if this is a TTS message                                | false                       |
 | file              | file contents                                                                                     | the contents of the file being sent                          | one of content, file, embed |
-| embed             | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object                                              | embedded `rich` content                                      | one of content, file, embed |
+| embeds            | array of [embed](#DOCS_RESOURCES_CHANNEL/embed-object) objects                                    | embedded `rich` content                                      | one of content, file, embed |
```

Comment:
can't fix in a suggestion but `embed` should be changed to `embeds` in the Required column